tests/WalletMgr.py

```diff
@@ -19,14 +19,22 @@ class WalletMgr(object):
 
     # pylint: disable=too-many-arguments
     # walletd [True|False] True=Launch wallet(keosd) process; False=Manage launch process externally.
-    def __init__(self, walletd, nodeosPort=8888, nodeosHost="localhost", port=9899, host="localhost"):
+    def __init__(self, walletd, nodeosPort=8888, nodeosHost="localhost", port=9899, host="localhost", clusterID=0):
         self.walletd=walletd
         self.nodeosPort=nodeosPort
         self.nodeosHost=nodeosHost
         self.port=port
         self.host=host
         self.wallets={}
         self.__walletPid=None
+        if clusterID != 0:
+            WalletMgr.__walletLogOutFile = "test_keosd_cluster" + str(clusterID) + "_out.log"
+            WalletMgr.__walletLogErrFile="test_keosd_cluster" + str(clusterID) + "_err.log"
+            WalletMgr.__walletDataDir="test_wallet_" + str(clusterID) 
+            if self.port == 9899:
+                self.port += clusterID     
```

Comment:
If separate keosd instances are going to run per cluster, then it would need to be allocated in the `port_type` `enum class` described above to use that port assignment scheme. Also, keosd could also use Unix sockets instead which would save a port type.